patches/chromium/fix_move_autopipsettingshelper_behind_branding_buildflag.patch

```diff
@@ -127,25 +138,31 @@ index 64152ef42a5225e857fde6982e4d88baf0c96341..86617d4ae181337b3558b18303f305d4
    AutoPipSettingHelper* get_setting_helper_for_testing() {
      return auto_pip_setting_helper_.get();
    }
+@@ -186,6 +189,7 @@ class PictureInPictureWindowManager {
+   // when a widget has been occluded by a video or document picture-in-picture
+   // window.
+   PictureInPictureOcclusionTracker* GetOcclusionTracker();
 +#endif
  #endif
  
    // Get the origins for initiators of active Picture-in-Picture sessions.
-@@ -237,7 +241,7 @@ class PictureInPictureWindowManager {
+@@ -243,7 +247,7 @@ class PictureInPictureWindowManager {
    // helper class for callbacks, to avoid re-entrant calls during pip set-up.
    static void ExitPictureInPictureSoon();
  
 -#if !BUILDFLAG(IS_ANDROID)
 +#if !BUILDFLAG(IS_ANDROID) && BUILDFLAG(GOOGLE_CHROME_BRANDING)
    // Create the settings helper if this is auto-pip and we don't have one.
    void CreateAutoPipSettingHelperIfNeeded();
- #endif  // !BUILDFLAG(IS_ANDROID)
-@@ -252,7 +256,9 @@ class PictureInPictureWindowManager {
+ 
+@@ -262,9 +266,11 @@ class PictureInPictureWindowManager {
  #if !BUILDFLAG(IS_ANDROID)
    std::unique_ptr<DocumentWebContentsObserver> document_web_contents_observer_;
  
 +#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
    std::unique_ptr<AutoPipSettingHelper> auto_pip_setting_helper_;
+ 
+   std::unique_ptr<PictureInPictureOcclusionTracker> occlusion_tracker_;
```

Comment:
@VerteDinde I don't see any chrome specific code involved in the tracker ? From the CL description looks like the primary user of the tracker would be pip settings UI, was that the motive to group them under this buildflag ?